style/gallery.less

```diff
@@ -148,9 +166,14 @@
 					margin: 16px;
 					font-size: 12px;
 					color: var( --gallery-primary-text-color );
-					overflow: hidden;
-					text-overflow: ellipsis;
-					white-space: nowrap;
+					
+					bdi {
```

Comment:
I don't think bdi should be used for styling. There must be a div or span with a meaningful class nearby.